mmv1/third_party/terraform/services/compute/data_source_google_compute_instance_guest_attributes_test.go.tmpl

```diff
@@ -0,0 +1,153 @@
+package compute_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"google.golang.org/api/compute/v1"
```

Comment:
```suggestion
```

Reply:
we need the compute for the "InstanceExists" check to go through. Will make it to beta instead of v1 for this.

The other 2 regarding Confidential config are unrelated. Got these fails in another PR, it's a "cannot find image" error